homeassistant/components/modbus/const.py

```diff
@@ -41,6 +41,7 @@
 CONF_RTUOVERTCP = "rtuovertcp"
 CONF_SCALE = "scale"
 CONF_SERIAL = "serial"
+CONF_STATE_CLASS = "state_class"
```

Comment:
Its still floating. I can add Modbus to use the global const PR once this is merged. Its this PR or the other, one has to be changed... but I think it would be nice to get this in 2021.9 even if the other doesn't make it - it doesn't really matter where this constant is saved for the feature to work.